docs/docs/howto/cluster-setup/bare-metal.md

```diff
@@ -189,7 +189,7 @@ helm repo add nvidia https://helm.ngc.nvidia.com/nvidia && helm repo update
 helm install --wait --generate-name \
    -n gpu-operator --create-namespace \
    nvidia/gpu-operator \
-   --version=v25.3.0 \
+   --version=v25.10.1 \
```

Comment:
Did you also update the GPU operator on the SNP CI machine to this version? If not, please do so, they should be in sync.